QuantLib/ql/time/schedule.hpp

```diff
@@ -90,11 +103,12 @@ namespace QuantLib {
         //@}
       private:
         bool fullInterface_;
```

Comment:
Where is the fullInterface() inspector? I commented out the `fullInterface_` variable and its use in 2 constructors and could successfully compile QuantLib and run the test-suite without problems. I suppose it's very trivial, though.